### PR TITLE
Fixes unicode and asteriks issues in by_searchable_text method

### PR DESCRIPTION
**Pull request by phgross**
* Fixes UnicodeEncodeError when searchable_text filter contains non ascii chars.
* Fixed asteriks handling in by_searchable_text query helper.

@deiferni please have a look ...

### CODE REVIEW

File: opengever/ogds/models/tests/test_query.py [hunk 1]
```diff
@@ -79,6 +79,22 @@ def test_by_searchable_handles_multiple_text_snippets(self):
             [self.hugo],
             User.query.by_searchable_text(['Br', 'go']).all())
 
+    def test_handles_no_ascii_characters_correctly(self):
+        self.assertEqual(
+            [self.james],
+            User.query.by_searchable_text([u'b\xf6nd']).all())
+
+    def test_handles_asteriks_correctly(self):
```
Comment (deiferni) [inline opengever/ogds/models/tests/test_query.py]: typo asteriks -> asterisk

**Comment by deiferni** [comment]
one small thing, rest looks :+1:

**Comment by phgross** [comment]
@deiferni updated ...

**Comment by deiferni** [comment]
thx!

**Pull request merged**